.cypress/integration/7_app_analytics.spec.js

```diff
@@ -372,11 +376,12 @@ describe('Viewing application', () => {
     cy.get('[data-test-subj="visualizeEditorRenderButton"]').click();
     cy.get('[data-test-subj="eventExplorer__saveManagementPopover"]').click();
     cy.get('[data-test-subj="eventExplorer__querySaveConfirm"]').click();
+    cy.wait(delay*5);
```

Comment:
necessary?

Reply:
Think its safe to remove, was a problem originally but seems to be working without.